src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/ServiceFilesMediator.kt

```diff
@@ -0,0 +1,79 @@
+package com.bardsoftware.papeeria.backend.cosmas
+
+import com.bardsoftware.papeeria.backend.cosmas.CosmasProto.*
+import com.bardsoftware.papeeria.backend.cosmas.CosmasGoogleCloudService.Companion.cemeteryName
+import com.bardsoftware.papeeria.backend.cosmas.CosmasGoogleCloudService.Companion.fileIdChangeMapName
+import com.bardsoftware.papeeria.backend.cosmas.CosmasGoogleCloudService.Companion.fileIdGenerationNameMapName
+import com.google.cloud.storage.Blob
+import com.google.cloud.storage.Storage
+import com.google.cloud.storage.StorageException
+import com.google.protobuf.GeneratedMessageV3
+import java.util.concurrent.ConcurrentHashMap
+import java.util.concurrent.locks.ReadWriteLock
+import java.util.concurrent.locks.ReentrantReadWriteLock
+
+class ServiceFilesMediator(private val storage: Storage, private val service: CosmasGoogleCloudService) {
```

Comment:
Some comments on the purpose of this class.

Reply:
done